docs/source/_json/types_document.resp

```diff
@@ -140,36 +140,18 @@ Content-Type: application/json+schema
     }, 
     "language": {
       "choices": [
-        [
-          "de", 
-          "Deutsch"
-        ], 
         [
           "en", 
           "English"
-        ], 
-        [
-          "es", 
-          "Espa\u00f1ol"
-        ], 
-        [
-          "fr", 
-          "Fran\u00e7ais"
         ]
       ], 
       "default": "en", 
       "description": "", 
       "enum": [
-        "de", 
-        "en", 
-        "es", 
-        "fr"
+        "en"
       ], 
       "enumNames": [
-        "Deutsch", 
-        "English", 
-        "Espa\u00f1ol", 
-        "Fran\u00e7ais"
+        "English"
```

Comment:
Hmm, where did those languages go? They should still include those for, those should be set as available and supported languages for all our tests. ([See testing.py](https://github.com/plone/plone.restapi/blob/5afe77b22bed7ca7f7d7680163866e69448bf4b0/src/plone/restapi/testing.py#L52-L71))